pkg/sql/row_converter.go

```diff
@@ -0,0 +1,210 @@
+// Copyright 2019 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License included
+// in the file licenses/BSL.txt and at www.mariadb.com/bsl11.
+//
+// Change Date: 2022-10-01
```

Comment:
I think we switched the header template to one that doesn't include the date.

Reply:
also this is pure copy/paste of code from somewhere else so I _think_ it is supposed to keep the copyright date from that file